.openshift-ci/jobs/push-images/registry_rw_login.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+registry_rw_login() {
+    if [[ "$#" -ne 1 ]]; then
+        die "missing arg. usage: registry_rw_login <registry>"
+    fi
+
+    local registry="$1"
+
+    case "$registry" in
+        docker.io/stackrox)
+            if [[ -z "$DOCKER_IO_PUSH_USERNAME" ]]; then
+                echo "DOCKER_IO_PUSH_USERNAME is not defined"
+                exit 0 # Change to exit 1 later
+            fi
+            docker login --username "$DOCKER_IO_PUSH_USERNAME" --password-stdin docker.io <<< "$DOCKER_IO_PUSH_PASSWORD"
+            ;;
+        quay.io/rhacs-eng)
+            if [[ -z "$QUAY_RHACS_ENG_RW_USERNAME" ]]; then
```

Comment:
Why are we only checking the username for existence? Can we either check both username and password or place a comment in the script specifying that username existence is enough?

Reply:
Done